README.md

```diff
@@ -99,6 +108,15 @@ Changes
 * Stop forcing a particular URL to work; users can now use whatever URL they
   please to store their Sumo tarballs.
 
+## v1.0.0
+
+* Release on github
+
+## v1.1.0
+
+* Add support for encrypted data bags
+* Add additional SumoLogic API log attribute to the HWRP
```

Comment:
Can you remove 1.0.0 and 1.1.1 from the doc?  It will be updated when the next version is released